curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/specify-upper-and-lower-number-of-matches.english.md

```diff
@@ -9,12 +9,20 @@ challengeType: 1
 Recall that you use the plus sign <code>+</code> to look for one or more characters and the asterisk <code>*</code> to look for zero or more characters. These are convenient but sometimes you want to match a certain range of patterns.
 You can specify the lower and upper number of patterns with <code>quantity specifiers</code>. Quantity specifiers are used with curly brackets (<code>{</code> and <code>}</code>). You put two numbers between the curly brackets - for the lower and upper number of patterns.
 For example, to match only the letter <code>a</code> appearing between <code>3</code> and <code>5</code> times in the string <code>"ah"</code>, your regex would be <code>/a{3,5}h/</code>.
-<blockquote>let A4 = "aaaah";<br>let A2 = "aah";<br>let multipleA = /a{3,5}h/;<br>multipleA.test(A4); // Returns true<br>multipleA.test(A2); // Returns false</blockquote>
+
+```js
+let A4 = "aaaah";
+let A2 = "aah";
+let multipleA = /a{3,5}h/;
+multipleA.test(A4); // Returns true
+multipleA.test(A2); // Returns false
+```
+
 </section>
 
 ## Instructions
 <section id='instructions'>
-Change the regex <code>ohRegex</code> to match only <code>3</code> to <code>6</code> letter <code>h</code>'s in the word <code>"Oh no"</code>.
+Change the regex <code>ohRegex</code> to match the phrase <code>"Oh no"</code> only when it has <code>3</code> to <code>6</code> letter <code>h</code>'s.
```

Comment:
```suggestion
Change the regex <code>ohRegex</code> to match the entire phrase <code>"Oh no"</code> only when it has <code>3</code> to <code>6</code> letter <code>h</code>'s.
```
I'm not sure if this is the perfect fix, but I think it needs to be made clear that 'no' needs matching too.  My first instinct when solving this was to use `/Oh{3,6}/` and wonder why that failed.

Particularly given that the examples and the seeds all use `regex.test(string)` and those would pass, even though the tests would fail.

Reply:
Does adding that make it precise enough?

Reply:
My suggestion?  I think it's a bit better. What do you reckon?

Reply:
I'll approve it regardless - it's a nice PR. I just wanted to bring it to your attention in case you had any inspiration!

Reply:
good enough for me